docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs crontabber.
+
+# Run crontabber sleeping 5 minutes between runs
+while true; do
+    python socorro/cron/crontabber_app.py
+    echo "Sleep 5 minutes..."
+    sleep 300
+done
```

Comment:
If we add a `set -e`, then the entire crontabber container will die if crontabber stops.

Interestingly, I just tweaked that in this PR:

https://github.com/mozilla-services/socorro/pull/3914/files#diff-9252dc07e44d2505f344c67d658b204eR27